nucliadb/nucliadb/ingest/settings.py

```diff
@@ -39,9 +39,7 @@ def _missing_(cls, value):
                 return member
 
 
-class Settings(BaseSettings):
-    grpc_port: int = 8030
-
+class DriverSettings(BaseSettings):
     # allowing defaults is not ideal
     # TODO: implement settings without defaults
```

Comment:
out of curiosity: why it is not ideal?

Reply:
IMO, things that require configuration like this should be explicit and hard fail.

For example, right now, with default redis, you also need to configure the redis to use but a user could leave this setting unfilled and wonder what is wrong when they get a bunch of redis errors.